Client/src/component/Announcements/PostingAnnouncements.tsx

```diff
@@ -0,0 +1,95 @@
+// import React from 'react'
```

Comment:
You should delete all the commented code before doing the pull request.
Incase you need to refer the code later you can visit previous commits.